habitat-baselines/habitat_baselines/config/experiments_hab3/pop_play_kinematic_oracle_humanoid_spot.yaml

```diff
@@ -30,6 +29,16 @@ defaults:
     - other_agent_gps
     - has_finished_oracle_nav
   - /habitat/task/actions@habitat.task.actions.agent_0_arm_action: arm_action
+  - /habitat/task/actions@habitat.task.actions.agent_0_rearrange_stop: rearrange_stop
+  - /habitat/task/actions@habitat.task.actions.agent_0_pddl_apply_action: pddl_apply_action
+  - /habitat/task/actions@habitat.task.actions.agent_0_oracle_nav_action: oracle_nav_action
```

Comment:
Why are you making all these changes to `pop_play_kinematic_oracle_humanoid_spot.yaml`? Will this affect ongoing experiments that use this config? Should we create our own config specifically for the HITL tool?

Reply:
This should only affect replicaCAD experiments, which we are not running for the paper. I can keep them here or put them in SIRo_dev

Reply:
Addressed here: https://github.com/facebookresearch/habitat-lab/pull/1510